tests/xtimer_msg_receive_timeout/main.c

```diff
@@ -30,27 +26,27 @@
 #include "xtimer.h"
 #include "timex.h"
 
-#define TEST_PERIOD (100000LU)
+#define TEST_PERIOD     (1*US_PER_SEC) /* 1 sec in US */
```

Comment:
Could you test on a device with a slower UART? e.g., arduino-mega2580? (Does the time needed for output even change the actual test timing?)